deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dhstore-data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 20Ti
```

Comment:
Max is 16 right?

Regardless, let's start with a small number and increase as needed.

Reply:
Good point - will decrease.

Reply:
Done